readycloud/readycloud.py

```diff
@@ -21,10 +21,19 @@ class ReadyCloud(object):
     :type token: str
     :param host: host with which you want to work (readycloud.com by default)
     :type host: str
+    :param api: api version (v2 by default)
+    :type host: str
     """
-    def __init__(self, token, host='https://readycloud.com/'):
+
+    API_v1 = 'v1'
+    API_v2 = 'v2'
+    ORG_PK = 1
```

Comment:
Org should not be predefined. To get a list of accessible orgs make a `GET /api/v2/orgs` call.